fe/fe-core/src/main/java/org/apache/doris/rewrite/FEFunctions.java

```diff
@@ -42,9 +42,6 @@
 import org.joda.time.DateTimeZone;
 import org.joda.time.LocalDateTime;
 
-import java.math.BigDecimal;
```

Comment:
Remove the above is better.
Could you please change the order of import as Ref [https://doris.apache.org/master/zh-CN/developer-guide/fe-eclipse-dev.html#import-%E9%A1%BA%E5%BA%8F]

Reply:
@EmmyMiao87  Done.